includes/managers/controls.php

```diff
@@ -970,7 +970,8 @@ public function get_teaser_template( $texts ) {
 				<div class="elementor-nerd-box-message"><?php echo $message; ?></div>
 			<?php }
 
-			if ( $texts['link'] ) { ?>
+			// Show a `Go Pro` button only if the user doesn't have Pro.
```

Comment:
Maybe add support for an experiment check. the method will receive the experiment name:
```
get_teaser_template( $texts, $experiment )
```
and if the experiment is off, it will show a message where to turn it on.

Reply:
?

Reply:
But what happens when the Pro is the one who registers the experiment, but the user uses an older Pro version with the new Core version?
He's gonna have a message to turn on the experiment even if he doesn't have Pro, or he has an old Pro version.